challenge.md

```diff
@@ -0,0 +1,9 @@
+// Think of a threat model. 
+
+
+Dear Co-Worker,
+
+As you know we work with highly sensitive data at DataCorp. As you have seen with other members of our market space we are a constant target for hackers around the world. Now it is especailly important to understand and address our vaulnerability.
```

Comment:
Heh, if only more DataCorp companies fully appreciated the sensitivity of their data.